File: CS/WindowsApplication1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraTab.Registrator;
using DevExpress.XtraTab.Drawing;
using DevExpress.XtraTab;

namespace WindowsApplication1
{
    public partial class Form1 : XtraForm
    {
        public Form1()
        {
            InitializeComponent();
            PaintStyleCollection.DefaultPaintStyles.Add(new MyRegistrator());
            xtraTabControl1.PaintStyleName = "MyStyle";
        }
    }
}

File: VB/WindowsApplication1/Form1.vb
Imports System.ComponentModel
Imports System.Drawing
Imports DevExpress.XtraEditors
Imports DevExpress.XtraTab.Registrator
Imports DevExpress.XtraTab.Drawing

Namespace WindowsApplication1

    Public Partial Class Form1
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
            Call PaintStyleCollection.DefaultPaintStyles.Add(New MyRegistrator())
            xtraTabControl1.PaintStyleName = "MyStyle"
        End Sub
    End Class
End Namespace
